app/lib/core/preferences.dart

```diff
@@ -65,6 +66,13 @@ class Preferences {
   /// default unit
   final TraleUnit defaultUnit = TraleUnit.kg;
 
+  /// default first day
+  final TraleFirstDay defaultFirstDay = TraleFirstDay.sunday;
+
+  /// default date format
+  final TraleDatePrintFormat defaultDatePrintFormat =
+      TraleDatePrintFormat.MMddyyyy;
+
```

Comment:
I guess the system default should be selected as default. Probably, most people are used to their local default.